src/Form/Fields.php

```diff
@@ -277,7 +278,7 @@ public function reset(): static
 		foreach ($this->data as $field) {
 			if ($field instanceof Field) {
 				$field->fillWithEmptyValue();
-			} else {
+			} elseif ($field instanceof InputField) {
 				$field->reset();
```

Comment:
Problem: This now leaves the HiddenField without being reset. Maybe check for `method_exists` instead?